website/content/en/docs/upgrading/v1-migration.md

```diff
@@ -176,233 +291,339 @@ kubectl patch customresourcedefinitions ec2nodeclasses.karpenter.k8s.aws -p "{\"
       --set controller.resources.limits.memory=1Gi \
       --wait
     ```
+    {{% alert title="Note" color="primary" %}}
+<!-- Note: don't indent this line to match the indenting of the alert box. Hugo will create a code block. -->
```

Comment:
lol this doesn't show up in the website, right? is it that there's no new line in between the last code block and this that this happens?

Reply:
Yeah, definitely doesn't show up it's a comment. IIUC a 4 space indent will get you a code block. I can give that a shot, but I think I had tried that.